day_6/exercises/person.js

```diff
@@ -7,5 +7,20 @@ Create a person class with at least 2 properties and 2 behaviors.  Call all
 person methods below the class so that they print their result to the
 terminal.
 */
+class Person {
+  constructor(hands,nose){
+    this.hands= hands;
+    this.nose = nose;
+  }
+  shakeHands (name){
+    console.log(name +" has " +this.hands + " hands");
+  }
+  smell(num){
```

Comment:
This `smell` method accepts an argument that is not used or modified or updated in any way. What's the purpose of the argument passed into this method?

Reply:
hmmmm, guess i can leave that blank?